modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -5482,13 +5482,20 @@ public CodegenParameter fromRequestBody(RequestBody body, Set<String> imports, S
             setParameterNullable(codegenParameter, codegenProperty);
         }
 
+        addJsonSchemaForBodyRequestInCaseItsNotPresent(codegenParameter, body);
+
         // set the parameter's example value
         // should be overridden by lang codegen
         setParameterExampleValue(codegenParameter, body);
 
         return codegenParameter;
     }
 
+    private void addJsonSchemaForBodyRequestInCaseItsNotPresent(CodegenParameter codegenParameter, RequestBody body){
+        if(codegenParameter.jsonSchema == null)
+            codegenParameter.jsonSchema = Json.pretty(body);
```

Comment:
I wonder if you can share a minimal spec to reproduce the issue fixed by this change?

Reply:
I commented on my phone, and this was attached at the PR level rather than response here. I'm duplicating my response here in case you didn't see it.

This affects every spec. You can see the current samples, none of which display body parameter information.